engines/base/src/main/resources/templates/test.html

```diff
@@ -34,15 +34,15 @@ <h2 th:text="${title}"></h2>
                         <option value="text/plain">txt</option>
                     </select></td></tr>
 
-                <th:block th:each="i: ${#numbers.sequence(0, T(java.lang.Math).min(18, transformOptions.size) - 1)}">
+                <th:block th:each="i: ${#numbers.sequence(0, T(java.lang.Math).min(18, transformOptions.size()) - 1)}">
```

Comment:
SpEL had issues without brackets, when it tries to reach through parameter name it seems to go through reflection that is limited by context settings, with brackets it just calls the method without issues.